docs/src/domains/steps/dns.md

```diff
@@ -1,170 +1,139 @@
-2. Follow the instructions on [how to set up a custom domain](./_index.md).
-   When you come to configuring your DNS provider, replace the suggested `CNAME` record with a record forwarding requests from {{<variable "YOUR_DOMAIN" >}} to `www.`{{<variable "YOUR_DOMAIN" >}}.
-
-The following DNS providers are known to support both domain forwarding and advanced DNS configurations:
-
-* [Namecheap](https://www.namecheap.com/support/knowledgebase/article.aspx/385/2237/how-to-redirect-a-url-for-a-domain/)
+1.  Make the `www.` version of your site the default (canonical) version
+    and configure your app and routes to [use the `www` subdomain as upstream](../../define-routes/_index.md).
 
+2.  Follow the instructions on [how to set up a custom domain](./_index.md).
+    When you come to configuring your DNS provider,
+    instead of a `CNAME` record, add a record forwarding requests
+    from {{<variable "YOUR_DOMAIN" >}} to `www.`{{<variable "YOUR_DOMAIN" >}}.
+ 
 <--->
-
+ 
 +++
-title=Use a `www` redirection service
+title=Use a redirection service
 file=none
 highlight=false
 +++
-
-If your registrar doesn't support custom records or domain forwarding you can consider using a redirection service.
-
-If your domain is `example.com`, a redirection service uses an `A` record to redirect all requests
-from `example.com` to `www.example.com`.
-
-One such redirection service is [WWWizer](http://wwwizer.com/naked-domain-redirect).
-
-To configure your domain name to point to your project using a redirection service:
-
-1. Make the `www.` version of your site the default (canonical) version and configure your app and routes to [use the `www` subdomain as upstream](../../define-routes/_index.md).
-2. Follow the instructions on [how to set up a custom domain](./_index.md).
-   When you come to configuring your DNS provider, replace the suggested `CNAME` record with
-   an `A` record pointing from your domain to the redirection service.
-   For WWWizer, that's the IP `174.129.25.170`.
-3. Ensure that your redirects use the same protocol:
+ 
+If your DNS provider doesn't support custom records or domain forwarding,
+consider using a redirection service.
+ 
+If your domain is `example.com`,
+a redirection service uses an `A` record to redirect all requests from `example.com` to `www.example.com`.
+ 
+To use a redirection service, follow these steps:
+ 
+1.  Make the `www.` version of your site the default (canonical) version
+    and configure your app and routes to [use the `www` subdomain as upstream](../../define-routes/_index.md).
+ 
+2.  Follow the instructions on [how to set up a custom domain](./_index.md).
+    When you come to configuring your DNS provider, instead of a `CNAME` record,
+    add an `A` record pointing from your domain to the redirection service.
+ 
+3.  Ensure that your redirects use the same protocol:
    `http://example.com` redirects to `http://www.example.com`, not to `https://www.example.com`.
```

Comment:
:question: Is it necessary to have `http` redirects for the redirection service? Because generally, `http` is automatically redirected to `https` and so `http` doesn't need to be included in the routes definition. If it does for redirects, that should be a separate step. If it's not necessary, I wouldn't use the example of `http`, since many people won't be defining that.